From: Ben Hutchings <ben@decadent.org.uk>
Date: Mon, 28 Oct 2013 03:55:02 +0000
Subject: perf: Avoid ABI change in 3.2.52

perf_cpu_context::active_pmu was renamed to perf_cpu_context::unique_pmu.
This is an API change but not an ABI change, so hide it from genksyms.

--- a/include/linux/perf_event.h
+++ b/include/linux/perf_event.h
@@ -934,7 +934,11 @@ struct perf_cpu_context {
 	int				exclusive;
 	struct list_head		rotation_list;
 	int				jiffies_interval;
+#ifdef __GENKSYMS__
+	struct pmu			*active_pmu;
+#else
 	struct pmu			*unique_pmu;
+#endif
 	struct perf_cgroup		*cgrp;
 };
 
